### ISSUE TITLE
Check column names of dataframes?

### ISSUE THREAD

**Issue by etiennebacher** [easystats/datawizard#276]
Should this kind of check be generalized to all other functions for consistency? If so, we could make a helper `.check_dataframe_names()` (we could also suggest using `janitor::clean_names()`).

https://github.com/easystats/datawizard/blob/89e81930548aecf37e87968f83e5007dd3c37b8f/R/adjust.R#L78-L83

**Comment by strengejacke** [comment]
Yes, sounds good! I'd also say that we can both suggest makenames and janitor.

**Comment by strengejacke** [comment]
Do we want this for _all_ other functions, or only where really needed?

**Comment by etiennebacher** [comment]
I guess all `data_` functions should have this. Any one of them could be called directly on a dataframe with bad column names and they almost have a `select` argument.
